src/bus/four_wire.rs

```diff
@@ -0,0 +1,87 @@
+#![allow(clippy::inconsistent_digit_grouping, clippy::unusual_byte_groupings)]
+
+use core::fmt;
+use embedded_hal::blocking::spi::{Transfer, Write};
+use embedded_hal::digital::v2::OutputPin;
+
+use crate::bus::Bus;
+
+const WRITE_MODE_MASK: u8 = 0b00000_1_00;
+
+// TODO This name is not ideal, should be renamed to VDM
```

Comment:
What stands VDM for?

Reply:
Variable length Data Mode, as defined in the data sheet.

![image](https://user-images.githubusercontent.com/607976/109526107-15bf2280-7a67-11eb-861f-1e15500eef44.png)